example/kotlinlib/linting/2-ktlint/build.mill

```diff
@@ -0,0 +1,69 @@
+package build
+
+import mill._, kotlinlib._
+import mill.util.Jvm
+import mill.api.Loose
+
+object `package` extends RootModule with KotlinModule {
+
+  def kotlinVersion = "1.9.24"
+
+  // TODO extract this as ktlint module, similar to CheckstyleModule
+  /**
+   * Runs [[https://pinterest.github.io/ktlint/latest/install/integrations/ ktlint]]
+   */
+  def ktlint(): Command[Unit] = T.command {
+    val exitCode = ktlint0()()
+
+    ktlintHandleErrors(exitCode)
+  }
+
+  private def ktlint0() = T.task {
+
+    T.log.info("running ktlint ...")
+
+    Jvm.callSubprocess(
+      mainClass = "com.pinterest.ktlint.Main",
```

Comment:
Does ktlint have any config file or other ways of configuring it? If so we should expose them